qless/jobs.py

```diff
@@ -0,0 +1,49 @@
+'''Generic utility jobs.'''
+
+import subprocess
+
+from . import logger
+
+
+class SubprocessJob(object):
```

Comment:
Yeah... that's when my spidey sense began to activate. I could see myself using this pretty regularly, but for the folks out there who decided to run the thing as `root`...

Any ideas of how else to distribute it? A separate package?